src/Cli/dotnet/commands/dotnet-workload/install/WorkloadInstallCommandParser.cs

```diff
@@ -61,6 +61,8 @@ internal static class WorkloadInstallCommandParser
 
         public static readonly Option<VerbosityOptions> VerbosityOption = CommonOptions.VerbosityOption();
 
+        public static readonly Option<string> FromRollbackFileOption = WorkloadUpdateCommandParser.FromRollbackFileOption;
```

Comment:
should it be hidden like it is in update?

Reply:
We're just reusing the same option here so the isHidden value will be true for both commands.